<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openButtonA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenSerialA</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabelA">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialA Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabelB">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialB Status</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portComboBoxA">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="baudComboBoxB">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="closeButtonA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseSerialA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButtonB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseSerialB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButtonB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenSerialB</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudComboBoxA">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="portComboBoxB">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="rotateStartButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RotateStart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateStopButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RotateStop</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusRotate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
